canary/producer-c/canary/KvsProducerSampleCloudwatch.cpp

```diff
@@ -366,7 +378,16 @@ INT32 main(INT32 argc, CHAR* argv[])
             }
 
             if (GETTIME() < runTill) {
+                frame.trackId = DEFAULT_VIDEO_TRACK_ID;
                 CHK_STATUS(putKinesisVideoFrame(streamHandle, &frame));
+
+                // Send frame on another track only if we want to run multi track. For the sake of
+                // multitrack, we use the same frame for video and audio and just modify the flags.
+                if (STRCMP(config.canaryTrackType, CANARY_MULTI_TRACK_TYPE) == 0) {
+                    frame.flags = FRAME_FLAG_NONE;
+                    frame.trackId = DEFAULT_AUDIO_TRACK_ID;
+                    CHK_STATUS(putKinesisVideoFrame(streamHandle, &frame));
```

Comment:
Question. What happens if we get an error on putframe (or other API calls for that matter here). Do we end up just logging it or we have a way to output error metrics? I presume the latter but wanted to confirm.

Reply:
You are right. PutFrame error metrics are emitted in the canary to cloudwatch.